tests/scripts/test_mob_merger_v3.py

```diff
@@ -11,8 +11,16 @@
 import sys
 
 # Import the functions from the script
-# You'll need to save the script as mob_merger_v3.py in the same directory
-from mob_merger_v3 import (
+# Instead of direct import, use relative import from the package
+import sys
+from pathlib import Path
+
+# Add the project root to Python path
+project_root = Path(__file__).parent.parent
+sys.path.insert(0, str(project_root))
```

Comment:
We shouldn't monkey patch the path, it tends to create fragile systems.

Pytest should add whatever is needed on the python path